### PR TITLE
i18n(es-ES): translated `tutorial-build a blog`

### PR DESCRIPTION
**Pull request by juanburg98**
Description the changes:

I traslated to spanish the section Tutorial >Build a Blog

Checklist:

 - [x] Introduction (100%)
 - [x] Setup (100%)
 - [x] Pages (100%)
 - [x] Components (100%)
 - [x] Layouts (100%)
 - [x] Astro API (100%)
 - [x] Islands (100%)

### CODE REVIEW

**Comment by Yan-Thomas** [comment]
Thanks so much for this @juanburg98, this is a gigantic contribution and we really appreciate it 💜. I believe our infrastructure isn't 100% ready for translated tutorials (still have to confirm this), will work on this next week - if that's the case -, so we can get your PR reviewed and merged! 🙌

**Comment by juanburg98** [comment]
Thanks to you Yan-Thomas, this is my first PR.
Do I leave this PR open or close it and upload it again later?

**Comment by Yan-Thomas** [comment]
No need to close @juanburg98! I'll convert it to a draft (so it isn't merged) before we verify everything on our end 🙌

**Comment by juanburg98** [comment]
obrigado irmão, cumprimentos do uruguai 💪💪🤝

File: src/content/docs/es/tutorial/4-layouts/2.mdx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+<h1> {frontmatter.title}</h1>
+<p>{frontmatter.pubDate.slice(0,10)}</p>
+<p><em>{frontmatter.description}</em></p>
+<p>Escrito por: {frontmatter.author}</p>
+<img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+<slot />
+```
+</Box>
+
+:::note[avoid duplication]
```
Comment (Yan-Thomas) [inline src/content/docs/es/tutorial/4-layouts/2.mdx:107]: Pointing out you forgot to translate this aside title! 🙌

**Comment by Yan-Thomas** [comment]
Forgot to point out this before @juanburg98, but there are a couple UI keys ([highlighted them here](https://github.com/withastro/docs/blob/main/src/i18n/en/ui.ts#L82-L94)) that you need to translate and add to the [`src/i18n/es/ui.ts`](https://github.com/withastro/docs/blob/main/src/i18n/es/ui.ts) for a few tutorial components (like the multiple choice and checklist) to be properly translated. Could you please add this to your PR? 🙌

**Comment by juanburg98** [comment]
Of course, I will work on the changes. 💪🤝

**Comment by juanburg98** [comment]
Fix bug in `layout` and add translations in `ui.ts` file

**Comment by juanburg98** [comment]
Fix the logo in the README

Before:
docs-README-md

After:
docs-README

**Pull request closed without merging**